### ISSUE TITLE
Build with XNNPACK

### ISSUE THREAD

**Issue by asus4** [asus4/tf-lite-unity-sample#36]
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/delegates/xnnpack

> XNNPACK is a highly optimized library of floating-point neural network inference operators for ARM, x86, and WebAssembly architectures in Android, iOS, Windows, Linux, macOS, and Emscripten environments. This document describes how to use the XNNPACK library as an inference engine for TensorFlow Lite.

**Comment by TheBricktop** [comment]
Could this boost the  performance?
I've tried mediapipe hands on gpu on android (s8) and they get about 20-25 fps but current hands implementation are very slow on pc. (i7 16 gb ram gti 960 ti)

**Comment by asus4** [comment]
Hi @TheBricktop, This is because the GPU delegate (a GPU acceleration feature of the TensorFlow Lite) is not supported on Window at the moment. I would like to support it eventually.

**Comment by TheBricktop** [comment]
What ones should do to implement it? Recompile tf lite library?

**Comment by asus4** [comment]
It looks like to be not officially supported.
https://github.com/tensorflow/tensorflow/issues/40325

**Comment by TheBricktop** [comment]
So on android the performance would be higher?

**Comment by asus4** [comment]
Yes, if you are interested in using MediaPipe without the GPU delegate, please refer XNNPACK(this issue) or [the integer quantized model](https://github.com/PINTO0309/PINTO_model_zoo/tree/master/033_Hand_Detection_and_Tracking)

**Comment by TheBricktop** [comment]
Would exporting tflite models to onnx and running in barracuda improve  the performance?

**Comment by asus4** [comment]
Yes, if all ops are supported in Barracuda, could improve it. Please refer to [the supported ops in Barracuda](https://docs.unity3d.com/Packages/com.unity.barracuda@1.0/manual/SupportedOperators.html)

**Comment by asus4** [comment]
Now, XNNPACK options are enabled in v2.4 libraries

**Comment by asus4** [comment]
Thanks, @tonysung

I mistakenly thought it would be automatically enabled on the CPU mode. I will add XNNPack delegate.

https://github.com/tensorflow/tensorflow/blob/3f878cff5b698b82eea85db2b60d65a2e320850e/tensorflow/lite/delegates/xnnpack/xnnpack_delegate.h#L48
